core/state/shared_pool.go

```diff
@@ -0,0 +1,36 @@
+package state
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"sync"
+)
+
+// sharedStorage is used to store maps of originStorage of stateObjects
+type SharedStorage struct {
+	poolLock   *sync.RWMutex
+	shared_map map[common.Address]*sync.Map
```

Comment:
"shared_map" needs to keep the same naming style with poolLock

Reply:
updated